lblod-blockchain-event-broker/endpoints/blockchain/blockchain.controller.js

```diff
@@ -37,32 +38,36 @@ const notify = async (req, res, next) => {
 };
 
 const validate = async (req, res, next) => {
-  const result = await sparQLService.getPublishResourcesByStatus(
+  const resultPublish = await sparQLService.getPublishResourcesByStatus(
+    STATUSES.PUBLISHED
+  );
+  const { bindings: publishedResources } = resultPublish.results;
```

Comment:
```suggestion
   const { results: { bindings: publishedResources } } =  await sparQLService.getPublishResourcesByStatus(
     STATUSES.PUBLISHED
   );
```
dit kan je zo ^ op 1 lijn schrijven